tests/testthat/test-dm-paste.R

```diff
@@ -52,4 +52,18 @@ test_that("generating code for creation of existing 'dm' works", {
     ),
     fixed = TRUE
   )
+
+  # produce code with colors
+  expect_output(
```

Comment:
Can you please use `verify_output()` ?

Reply:
weirdly the last example leads to an error when used in `verify_output()`, even though it works fine otherwise. It would be nice if you could have a look.

Reply:
Relevant commit is 374d7c6